test/VerifySignedMessage.test.js

```
@@ -0,0 +1,433 @@
+      signer: '0xf354C24DCE0f33E73C2150B5fC39B2d676a39a8f',
+      accountAddress: '0x044e9527429b01211b06B06885E6c5873f82156C',
+      functionName: 'metaDelegateCall',
+      signedParams: [
+        {
+          name: 'to',
+          type: 'address',
+          value: '0x53D468E719694f3e542Dda96a237Af08eb394f2C'
+        },
+        {
+          name: 'data',
+          type: 'bytes',
+          value: '0x743f29da00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000ff20817765cb7f73d4bde2e66e067e58d11095c20000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000083401b9c38effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
+          callData: {
+            functionName: 'tokenToEth',
+            params: [
+              { name: 'bitmapIndex', type: 'uint256', value: '0' },
+              { name: 'bit', type: 'uint256', value: '16384' },
+              {
+                name: 'token',
+                type: 'address',
+                value: '0xfF20817765cB7f73d4bde2e66e067E58D11095C2'
+              },
+              {
+                name: 'tokenAmount',
+                type: 'uint256',
+                value: '1000000000000000000'
+              },
+              { name: 'ethAmount', type: 'uint256', value: '9019460273038' },
+              {
+                name: 'expiryBlock',
+                type: 'uint256',
+                value: '115792089237316195423570985008687907853269984665640564039457584007913129639935'
+              },
+              { name: 'to', type: 'address' },
+              { name: 'data', type: 'bytes' }
+            ]
+          }
+        }
+      ]
+    }
+    brink.verifySignedMessage(message)
+  })
+
+  it('Expects missing param in signed tokenToEth message', async function () {
```

test wording nitpick .. these should be worded as `should return an error when ...` or something that describes the expected behavior